### PR TITLE
JAMES-2349 Fix user quota endpoints documentation

### PR DESCRIPTION
**Pull request by aduprat**

### CODE REVIEW

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
test this please

File: server/protocols/webadmin/webadmin-mailbox/src/main/java/org/apache/james/webadmin/routes/UserQuotaRoutes.java [hunk 1]
```diff
@@ -160,7 +164,8 @@ public void defineDeleteQuotaSize() {
     })
     @ApiResponses(value = {
             @ApiResponse(code = HttpStatus.NO_CONTENT_204, message = "OK. The value has been updated."),
-            @ApiResponse(code = HttpStatus.BAD_REQUEST_400, message = "The body is not a positive integer."),
+            @ApiResponse(code = HttpStatus.BAD_REQUEST_400, message = "The body is not a positive integer or -1."),
```
Comment (rouazana) [inline server/protocols/webadmin/webadmin-mailbox/src/main/java/org/apache/james/webadmin/routes/UserQuotaRoutes.java]: s/or -1/nor -1 for unlimited/ ?

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**